xir/program.py

```diff
@@ -24,6 +24,8 @@
 Param = Union[complex, str, Decimal, DecimalComplex, bool, List["Param"]]
 Params = Union[List[Param], Dict[str, Param]]
 
+ARBITRARY_NUM_WIRES = "..."
```

Comment:
We wouldn't need a constant like this if we just used `...` directly, right?

Reply:
see my comment above https://github.com/XanaduAI/xir/pull/6/files#r762130077

Reply:
Okay, let's continue this discussion in https://github.com/XanaduAI/xir/pull/6#discussion_r769687832.

Reply:
I'm not sure why I cannot reply to the other discussion thread (there simply is no reply-box!), so I'll just write here instead. 🤷 

> That's true, but I would still argue that, as a user of the XIR, it is more intuitive to deal with `...` objects then a string constant which is meant to represent a similar thing. Also, I wouldn't really expect the representation of the "arbitrary number of wires" object to change so there isn't much need to design our code around that use case.

I'm leaning towards agreeing with @Mandrenkov here, since we'll likely not update this more often than any of any other syntax. This would be closer to how we handle e.g., the inverse and ctrl modifiers.